core/frontend/vitest.config.mts

```diff
@@ -0,0 +1,68 @@
+import { coverageConfigDefaults, defineConfig } from 'vitest/config';
+import { viteStaticCopy } from 'vite-plugin-static-copy';
+
+export default defineConfig({
+  test: {
+    dir: "src",
+    setupFiles: "./src/test/setupTests.ts",
+    // include: ["**/<testfileName>.test.ts"], // to adhere to chai's describe.only/it.only
+    browser: {
+      provider: "playwright",
+      enabled: true,
+      name: "chromium",
+      headless: true,
+      screenshotFailures: false
+    },
+    coverage: {
+      provider: "v8",
+      include: [
+        "src/**/*"
+      ],
+      exclude: [
+        ...coverageConfigDefaults.exclude,
+        "src/test/**/*",
+        "**/*.d.ts",
+        "**/*.d.tsx"
+      ],
+      reporter: [
+        "text-summary",
+        "lcov",
+        "cobertura"
+      ],
+      reportsDirectory: "./lib/cjs/test/coverage",
+    },
+    pool: "threads",
+    fileParallelism: false // Had to disable parallel test runs due to Worker related tests timing out and not fetching properly.
+  },
+  plugins: [
+    viteStaticCopy({
+      targets: [
+        {
+          src: 'lib/test/test-worker.js',
+          dest: '.'
+        },
+        {
+          src: 'lib/public/*',
+          dest: '.'
+        },
+        {
+          src: 'src/test/public/*',
+          dest: '.'
+        }
+      ]
+    })
+  ],
+  resolve: {
+    alias: {
+      "../../package.json": "../package.json",
+    }
+  },
+  optimizeDeps: {
+    include: ['@itwin/core-frontend', "@itwin/core-common", "@itwin/core-bentley"],
+  },
+  build: {
+    commonjsOptions: {
+      include: [/@itwin\/core-frontend/, /@itwin\/core-common/, /@itwin\/core-bentley/, /node_modules/],
```

Comment:
same comment about core frontend

Reply:
yeah, this did not make sense, with us using the ESM versions of the packages... removed build.commonjsOptions, it's something I pasted from DTA's vite config